Setup/Patch/Data/RebuildReplicasPatch.php

```diff
@@ -67,10 +62,19 @@ public function apply(): PatchInterface
         $storeIds = array_keys($this->storeManager->getStores());
         // Delete all replicas before resyncing in case of incorrect replica assignments
         foreach ($storeIds as $storeId) {
+            if (!$this->algoliaCredentialsManager->checkCredentialsWithSearchOnlyAPIKey($storeId)) {
+                $this->logger->warning("Algolia credentials are not configured for store $storeId. Aborting replica rebuild patch. If you need to rebuild your replicas run `bin/magento algolia:replicas:rebuild`");
```

Comment:
```suggestion
                $this->logger->warning("Algolia credentials are not configured for store $storeId. Skipping auto replica rebuild for this store. If you need to rebuild your replicas run `bin/magento algolia:replicas:rebuild`");
```
Just a little tweak to the warning message language I think is warranted here.